forge/test/operators/pytorch/eltwise_unary/test_unary.py

```diff
@@ -160,6 +165,31 @@ class TestParamsData:
     test_plan_implemented: TestPlan = None
     test_plan_not_implemented: TestPlan = None
 
+    no_kwargs = [
+        None,
+    ]
+
+    kwargs_clamp = [
+        {"min": 0.0, "max": 0.5},
+        {"min": 0.5, "max": 0.0},
+        {"min": 0.2},
+        {"max": 0.2},
+    ]
+
+    kwargs_pow = [
+        {"exponent": 0.5},
```

Comment:
do we test negative exponents?

Reply:
Currently we don't. Should we include negative exponents?

Reply:
I would, that is the half of the domain of legal exponents.

Reply:
Also I would test non integer exponents. torch.pow supports non integer exponents. If we don't support them, we should be aware. I would test something like 1.5, 1.2...

Reply:
Added mentioned exponent values.